docs/azmcp-commands.md

```diff
@@ -29,6 +29,12 @@ azmcp server start [--transport <transport>]
 azmcp subscription list [--tenant-id <tenant-id>]
 ```
 
+### Best Practices
```

Comment:
Should this new tool also be on the root README.md with the other tools?  Or do we prefer to intentionally not list it?

Reply:
added to README